website/en/docs/lang/subtypes.md

```diff
@@ -113,22 +113,32 @@ recursively until we can decide if we have a subtype or not.
 
 #### Subtypes of functions <a class="toc" id="toc-subtypes-of-functions" href="#toc-subtypes-of-functions"></a>
 
-Flow compares two functions by comparing its inputs and outputs. If all the
-inputs and outputs are a subset of the other function, then it is a subtype.
+The subtyping of functions is a bit trickier. So far we've used the rule that one
```

Comment:
Subtyping rules for functions are a bit more complicated. So far, we've seen that `A` is a subtype of `B` if `B` contains all the possible values in `A`. For functions, it's not clear how this relationship would apply. To simplify things, you can think of a function type `A` being a subtype of a function type `B` if functions of type `A` can be used wherever a function of type `B` is expected.